jscomp/bsb/bsb_build_util.ml

```diff
@@ -22,6 +22,8 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
+open! Ext 
```

Comment:
Why `open!`?

Reply:
https://stackoverflow.com/questions/22607755/what-does-open-mean-in-ocaml